Developer.md

```diff
@@ -0,0 +1,22 @@
+# Building flyxc
+
+## Preparation
+
+Copy `keys.ts.dist` to `keys.ts` and fill it out (there are two such files)
+
+## Installation of dependencies
+`npm install --no-fund --no-audit` To install all dependencies (possible to use also yarn: `yarn install`)
+
+
+## Building flyxc
+
+`npx nx run-many -t build`
+
+## Starting server
+
+`npm run dev`
+
+# FAQ
+
+1) Cannot find name 'gtag'
```

Comment:
Have you had issue with that?
I added some commits related to gtag and maybe you didn't `npm i` after you pulled the changes?